types/xml-encryption/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for xml-encryption 1.2
+// Project: https://github.com/auth0/node-xml-encryption#readme
+// Definitions by: Chris Midgley <https://github.com/midgleyc>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+export interface EncryptOptions {
+    rsa_pub: string | Buffer;
+    pem: string | Buffer;
+    keyEncryptionAlgorithm: string;
+    encryptionAlgorithm?: string;
```

Comment:
this is closed list, can this be converted to aliases?:
```ts
encryptionAlgorithm?: EncryptionAlgorithm;
```
```ts
type EncryptionAlgorithm = 'http://www.w3.org/2001/04/xmlenc#aes128-cbc' | '...' | ....;
```
https://github.com/auth0/node-xml-encryption/blob/master/lib/xmlenc.js#L99

Reply:
yes, good spot, thanks